.circleci/config.yml

```diff
@@ -123,6 +123,21 @@ jobs:
             go mod init tmp &&
             go get golang.org/x/exp/cmd/gorelease@latest
       - run: gorelease -base=v2.0.0
+  proto_lint:
+    docker:
+      - image: bufbuild/buf:0.37.0
+    steps:
+      - checkout
+      - run: buf build
+      - run: buf lint
+      - run: buf breaking --path protoc-gen-openapiv2/options/annotations.proto --path protoc-gen-openapiv2/options/openapiv2.proto --against 'https://github.com/grpc-ecosystem/grpc-gateway.git#branch=master'
```

Comment:
These two protofiles are our only public dependencies, so they're the only ones we really care about making sure we don't break.